examples/chat-postgres/README.md

```diff
@@ -0,0 +1,38 @@
+# Next.js/Postgres Chat Example
```

Comment:
Could you add a README here, based on the other readmes in the examples/ directory?

Reply:
Done